src/h3lib/include/h3api.h.in

```diff
@@ -152,46 +163,47 @@ typedef struct {
  */
 /** @brief find the H3 index of the resolution res cell containing the lat/lng
  */
-H3Index H3_EXPORT(geoToH3)(const GeoCoord *g, int res);
+DECLSPEC H3Index H3_EXPORT(geoToH3)(const GeoCoord *g, int res);
```

Comment:
Not going to block on this, but I wonder if it would have been possible to modify the `H3_EXPORT` instead to tackle this, eg: `H3_EXPORT(H3Index, geoToH3)(const GeoCoord *g, int res);` and macro args may or may not include the DECLSPEC thing.